src/calendar/header/index.tsx

```diff
@@ -102,7 +102,8 @@ const CalendarHeader = forwardRef((props: CalendarHeaderProps, ref) => {
     importantForAccessibility,
     numberOfDays,
     current = '',
-    timelineLeftInset
+    timelineLeftInset,
+    accessibilityRole
```

Comment:
Where this prop is declared?

Reply:
defined in the props. Did you mean that?